### PR TITLE
Adding Shipwire_connector and Update podio_connector

### PR DESCRIPTION
**Pull request by Adamwgoh**
Added 1 trigger and 2 actions : 

Triggers
New or updated order 

Actions
Get product details 
Search products 

This is for AscentERP. Reference issue https://github.com/workato/workato/issues/13587

### CODE REVIEW

File: basic_auth/shipwire_connector.rb [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+          {	name:	'dimensions', type: :object, properties: [{	name: 'resourceLocation'}]},
+          {	name: 'values', type: :object, properties: [{name: 'resourceLocation'}]},
+          {	name: 'alternateNames', type: :object, properties: [{	name: 'resourceLocation'}]},
+          {	name: 'technicalData', type: :object, properties: [{	name: 'resourceLocation'}]},
+          {	name: 'flags', type: :object, properties: [{	name: 'resourceLocation'}]},
+          {	name: 'enqueuedDimentions', type: :object, properties: [{	name: 'resourceLocation'}]},
+          {	name: 'innerPack', type: :object, properties: [{	name: 'resourceLocation'}]},
+          {	name: 'masterCase', type: :object, properties: [{	name: 'resourceLocation'}]},
+          {	name: 'pallet', type: :object, properties: [{	name: 'resourceLocation'}]},          
+#          { name: 'classification'},
```
Comment (eeshan92) [inline basic_auth/shipwire_connector.rb]: Do we need this? If not, remove this field.

Comment (Adamwgoh) [inline basic_auth/shipwire_connector.rb]: Removed 'classification'

File: basic_auth/shipwire_connector.rb [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+            {	name: 'resourceLocation'},
+            {	name: 'resource', type: :object, properties: object_definitions['product']}
+        	]
+      }
+    }
+  },
+
+  triggers: {
+
+		new_or_updated_order: {
```
Comment (eeshan92) [inline basic_auth/shipwire_connector.rb]: Can we use descending trigger for this?

File: oauth2/podio_connector.rb [hunk 3]
```diff
@@ -15,9 +15,9 @@
         'https://podio.com/oauth/token'
       },
 
-      client_id: 'YOUR_PODIO_CLIENT_ID',
+      client_id: 'workato-wwzue9',
```
Comment (eeshan92) [inline oauth2/podio_connector.rb]: should hide credentials

**Comment by eeshan92** [comment]
https://github.com/workato/workato/issues/13980

File: oauth2/podio_connector.rb [hunk 4]
```diff
@@ -27,50 +27,139 @@
...
+           { name: 'Application ID', optional: 'false'}
+         ]
+      	},
+
+	execute: ->(connection, input) {
+	   { tags: get("https://api.podio.com/tag/app/#{input['Application ID']}")}
+	},
+
+	output_fields: ->(object_definitions){
+	   object_definitions['tag']
```
Comment (eeshan92) [inline oauth2/podio_connector.rb]: Would this be an array instead?

Comment (Adamwgoh) [inline oauth2/podio_connector.rb]: Yes. My bad to miss that. Changing it.

File: basic_auth/shipwire_connector.rb [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+          { name: 'countryOfOrigin'},
+          { name: 'creationDate', type: :timestamp},
+          { name: 'archivedDate', type: :timestamp},
+          { name: 'status'},
+          { name: 'storageConfiguration'},
+          { name: 'batteryConfiguration'},
+          { name: 'itemCount'},
+          { name: 'vendorID'},
+          { name: 'vendorExternalId'},
+          { name: 'dimensions', type: :object, properties: [{ name: 'resourceLocation'}]},
```
Comment (yogeshpadmanabhan) [inline basic_auth/shipwire_connector.rb]: Why is the ```resource``` property not included for any of the object types? Looks like that has the actual data.

**Pull request merged**